usb.cpp

```diff
@@ -96,12 +96,14 @@ namespace bsp
     void usbDeinit()
     {
         // Restart HW tick for resume operation
-        xTimerStart(usbTick, 1000);
+        xTimerStart(usbTick, 0);
```

Comment:
What is the point of such an operation? Do we want here to fire the timer immediately?

Reply:
The second argument is not the delay of the timer start, but xBlockTime, that is the time, in ticks, that the calling task should be held in the Blocked state to wait for the start command to be successfully sent to the timer command queue.

Reply:
Then, is it possible for `xTimerStart` request not reaching destination? Software timers are managed by the idle task which has its own requests queue.

Reply:
Unlikely, but yes, it is possible. FreeRTOS timers are handled in the `Tmr Svc` thread.

Reply:
At least, handle return value :)

Reply:
@swichu91 done